apps/src/sites/studio/pages/levelbuilder.js

```diff
@@ -49,6 +49,15 @@ window.levelbuilder.copyWorkspaceToClipboard = function() {
   copyToClipboard(str);
 };
 
+window.levelbuilder.copySelectedBlockToClipboard = function() {
+  if (Blockly.blockly_.selected) {
```

Comment:
Can you add `selected` as a read-only property here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/blockly/cdoBlocklyWrapper.js (similar to https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/blockly/googleBlocklyWrapper.js#L142), then change this line to just `Blockly.selected`?